### PR TITLE
Production merge

### PR DESCRIPTION
**Pull request by Luffytaro22**
> # Javascript Capstone Project 💯 
> **In this PR we are merging all the development features from the Dev branch to the production/master branch. The features added are as follows:**
> * `Fetch` information about TV Shows from `TVMaze API`.
> * The information is displayed dynamically with JavaScript.
> * Each TV show has a `like`, `comment`, and `reservation` buttons.
> * Use of the `Involvement API` developed by Microverse to add and retrieve data about likes, comments, and reservations.
> * The `comments` and `reservations` pop up when the user clicks on each button.
> * The info about likes, comments, and reservations is displayed when the page charges.
> * Created `counters` functions to display the `number` of TV shows on the main page, and the number of comments and reservations made.
> * Each counter function was `tested` using `Jest` and `jsdom`.
> * The application is `responsive`, so it adapts perfectly to different screen sizes.

### CODE REVIEW

File: src/style.css [hunk 1]
```diff
@@ -0,0 +1,453 @@
...
+    display: none;
+  }
+
+  #hamburguer.hide {
+    display: block;
+  }
+
+  #hamburguer {
+    display: none;
+  }
```
Comment (OyePriscilla) [inline src/style.css]: - [x] Kindly avoid code repetition in order to follow CSS best practises. 👍 You nicely did the needfull in line 374. 👏

File: README.md [hunk 2]
```diff
@@ -47,187 +18,141 @@ After you're finished please remove all the comments and instructions!
...
+- **Fetchs Likes, Comments and Reservations from an Involvement API**
+- **Comment and Reservations PopUp Deployed**
+- **Item, Like, Comment and Reservation Counter Deployed**
+- **All Counters Tested with Jest**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- Video presentation -->
+## Video Presentation <a name="video-presentation"></a>
+[Click here](https://drive.google.com/file/d/1yALLexsypFizmaAbXW6Rr8J1zhxZVQnp/view?usp=sharing)
```
Comment (OyePriscilla) [inline README.md]: - [x] Clear and engaging presentation, kindly highlight some interesting piece of code or something you built that you are very proud of in accordance to the project requireent. 👍

File: src/items-counter.test.js
Comment (OyePriscilla) [inline src/items-counter.test.js:1]: - [optional] Your file organization would be more professional if you can group all the test files in a directory. ℹ️

**Review by OyePriscilla — CHANGES_REQUESTED** [review]
Hi @Luffytaro22, @Gh-Reza and @demesameneshoa 👍 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

so-close

## Highlights
Used JavaScript to make websites dynamic and build basic single page apps ✔️ 
Used ES6 modules ✔️
Used webpack ✔️
Wrote units tests for a JavaScript app ✔️
Sent and received data from an API ✔️
Implemented JavaScript code asynchronous ✔️
Performed a code review for a team member ✔️

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/style.css (hunk 1 above)
Comment (demesameneshoa) [reply src/style.css]: @OyePriscilla  these css codes are for different breakpoints of various screens, they are not repetitions.

Comment (demesameneshoa) [reply src/style.css]: Done 💯

**Review by frank1738 — APPROVED** [review]
Hi Team, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 




## Highlights

✅ Correct use of Gitflow
✅ No linter error
✅ Good Design👍 

## Optional suggestions

N/A

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**